onnx_chainer/functions/array.py

```diff
@@ -316,3 +316,37 @@ def convert_Where(func, opset_version, input_names, num_outputs, context,
                   parameters):
     input_names.insert(0, context.get_name(func.condition))
     return onnx_helper.make_node('Where', input_names, num_outputs),
+
+
+@support((7, 9))
+def convert_Repeat(func, opset_version, input_names, num_outputs, context,
+                   parameters):
+    repeats = func.repeats
+    if len(repeats) > 1:
+        raise NotImplementedError(
+            'ONNX-Chainer currently does not support elementwise repeat')
+
+    gb = onnx_helper.GraphBuilder()
+    inputs = list(input_names)
+    axis = func.axis
+    if axis is None:
+        shape_param = chainer.Parameter(np.array([-1]))
+        parameters.append(shape_param)
+        input_names.append(context.get_name(shape_param))
+        inputs = [gb.op('Reshape', input_names)]
+        scales = [float(repeats[0])]
+    else:
+        scales = [1.0] * func.inputs[0].data.ndim
+        scales[axis] = float(repeats[0])
+
+    if opset_version == 7:
+        gb.op('Upsample', inputs, scales=scales)
+        return gb.nodes()
+
+    if opset_version == 9:
```

Comment:
JFYI: Note `Upsample` is deprecated at opset 10. `Resize` is the new one.

Reply:
thanks, we will support opset=10 later